theme/assets/sass/components/_type-specimen.scss

```diff
@@ -99,122 +131,106 @@
 
 #specimen-type-scale-condensed {
     .docs-scale-sample-title-2xl {
-        font-size: map.get($theme-type-scale-condensed-firefox, 'title-xl-size');
-        line-height: map.get($theme-type-scale-condensed-firefox, 'title-xl-line-height');
+        font-size: 24px;
```

Comment:
**curious**: I guess this is because we do things a bit different for the theme folder? no lib/theme to draw from for the values here?